### ISSUE TITLE
electron repulsion integral eri(k) of a khf object

### ISSUE THREAD

**Issue by hungpham2017** [sunqm/pyscf#156]
Hello Qiming,

I am trying to look for the **eri(k)** of **a khf object** or equivalently its direct-space counterpart **eri(G0)** (i.e. for a unit cell).
I understand that for a Gamma-point calculation with pbc.hf module, _eri can be stored or computed with the kmf.with_df.get_ao_eri() function.
However, I could not find something similar in the khf module. As far as I understand, in the khf module the vj, vk are computed numerically, not directly from eri and 1-RDM. 
**_Is there any way (analytical or numerical) to evaluate the **eri(k)** or **eri(G0)** of the reference unit cell?_**

_My purpose:_ I am testing the evaluation of total energy of a k-point sampling calculation in direct space. I was able to do it if 1-RDM, hcore, vhf are transformed into direct space. Now I would like to have **eri(k)** and wanna transform it to direct space. Thus, I will be able to directly compute the vhf in direct-space as.
 
Thank you very much.
Hung

**Comment by gkc1000** [comment]
They are computed with density fitting; see eg

https://github.com/sunqm/pyscf/blob/master/pyscf/pbc/ao2mo/eris.py

**Comment by hungpham2017** [comment]
Thank you for your response. 
I thought that module was not what I was looking for since I got wrong the results of Coulomb potential at a k-point using that. It turns out I did not sum over the contribution of orbitals at other k-points.
Also, it is actually was well-written in the manual as well. [(http://sunqm.github.io/pyscf/pbc/df.html#index-eri-tensor-and-integral-transformation)  ](url)

Thank you again for your help.
Hung
